docs/developer-guide/iso-installer.md

```diff
@@ -0,0 +1,49 @@
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+
+<!--
+    Copyright 2021, Joyent, Inc.
+-->
+
+# Triton ISO Installation
+
+If one wishes to boot off a ZFS pool, one can use the Triton ISO installer to
+boot once off of DVD-ROM, and on subsequent boots boot off of a bootable
+`zones` ZFS pool.
+
+The ISO installer can be used in a virtual machine environment, in lieu of
+[CoaL](./docs/developer-guide/coal-setup.md), as well.  Be sure to download
+the [ISO
+image](https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/iso-latest.iso)
+and follow the main README. 
+
+## Minimum Requirements
+
+The ISO installer requires more disk space than a USB-key Triton deployment,
+but is otherwise the same.
+
+- Intel Processors with VT-x and EPT support (all Xeon since Nehalem), or AMD Processors no earlier than EPYC or Zen. 
+- 16 GB RAM
+- 60 GB available storage
```

Comment:
Same question as above. Also why the difference between recommended disk sizes?

Reply:
I was being paranoid about both images AND about having sufficient swap for 16GB of RAM (our swap algorithm does 1-for-1 there until sufficiently large RAM).

Reply:
Oh and also it needs room for Platform Images.